src/services/walletConnect/index.ts

```diff
@@ -1,11 +1,11 @@
-import { SignerPayloadJSON } from '@polkadot/types/types';
+import { Signer } from '@polkadot/types/types';
 import { WalletAccount } from '@talismn/connect-wallets';
 import type { SessionTypes } from '@walletconnect/types/dist/types/sign-client/session';
 import UniversalProvider from '@walletconnect/universal-provider';
 import logo from '../../assets/wallet-connect.svg';
 import { config } from '../../config';
 
-// TODO: improve this
+// TODO: improve (missing methods)
```

Comment:
Is this referred to the methods tagged as 'Unused' ? Do we need them? Is yes, I agree with the TODO, otherwise I would remove it.
```
        getAccounts: () => Promise.resolve([]), // Unused
        subscribeAccounts: () => undefined, // Unused
        transformError: (err: Error) => err, // Unused
```